:fire: Added archish27 nora bot project (#144)
Codesee-io/oss-port: projects/Archish27/nora-covid-19-bot.mdx

---

> name: Nora Covid-19 Bot

Reviewer: ```suggestion
name: Nora COVID-19 Bot
```

repoUrl: https://github.com/Archish27/nora-covid-19-bot

> description: Coronavirus Self Awareness Bot that addresses typical questions about the COVID-19 virus to help you to know about corona virus. It provides awarness about Covid-19 social distancing, self isolation, quarantine.

Reviewer: ```suggestion
description: COVID-19 self-awareness bot that addresses frequently asked questions to help educate folks about the virus. It provides awareness about social distancing, self-isolation, quarantine.
```

languages: # A list of programming languages & frameworks used in this project
  - Python
  - Rasa
tags: # A list of keywords relevant to the contributors for this project
  - First Timer Friendly
  - Health

>   - Social Activism

Reviewer: ```suggestion
  - Social Activism
  - Machine Learning 
```

avatar: logo.png
websiteUrl: https://norabot.netlify.app
currentlySeeking:

>   - Machine Learning Developers

Reviewer: ```suggestion
  - Developers
```

  - Backend Developers
  - Maintainers
featuredMap:
  url: https://app.codesee.io/maps/public/e1e6eca0-21fe-11ec-9348-e1d61854fcda
  description: CodeSee Map to help you get started abd get to know the project
contributionOverview:
  mainLocation: India
  idealEffort: 1 PR a month
  isMentorshipAvailable: false
learnLinks: # (optional) A list of links to support new contributors in learning about your project and tech stack and culture / community
  - title: Getting Started with Rasa
    url: https://rasa.com/docs/
---

<!-- Divide your content into sections using our handy shortcodes. Each section is optional -->

<Overview>

<img src="https://raw.githubusercontent.com/Archish27/nora-covid-19-bot/master/logo-white.png" alt="logo" width="80px"/>

<h3 align="center">
  Nora COVID-19 Self Awareness Bot
</h3>

> ## About project

Reviewer: ```suggestion
## About 
```
